CDP4Reporting/CDP4Reporting.csproj

```diff
@@ -0,0 +1,339 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <ProjectGuid>{C31FF150-E45C-4136-96CD-58FBF9B00C08}</ProjectGuid>
+    <TargetFramework>net452</TargetFramework>
+
+    <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\</SolutionDir>
+    <AssemblyTitle>CDP4Reporting</AssemblyTitle>
+    <Company>RHEA System S.A.</Company>
+    <Product>CDP4Reporting</Product>
+    <Description>CDP4 Reporting Plugin</Description>
+    <Copyright>Copyright © RHEA System S.A.</Copyright>
+    <AssemblyVersion>6.0.0.0</AssemblyVersion>
```

Comment:
Verions should be 0.0.1.0 because it is not finished yet. 
Reporting will probably ship as a alfa/beta version with the next IME release.